### PR TITLE
Add ePayment checklist

### PR DESCRIPTION
**Pull request by rebekaburnett**
@tht13 - This is a draft.
It's based on the ecom and checkout checklists.   
I need something to link the "callbacks" to.

### CODE REVIEW

File: vipps-epayment-api-checklist.md [hunk 1]
```diff
@@ -1,6 +1,113 @@
...
+  - [ ] `Vipps-System-Name`
+  - [ ] `Vipps-System-Version`
+  - [ ] `Vipps-System-Plugin-Name`
+  - [ ] `Vipps-System-Plugin-Version`
+- [ ] Follow the [orderId recommendations](https://vippsas.github.io/vipps-developer-docs/docs/vipps-developers/common-topics/orderid).
+- [ ] We recommend using Vipps Order management, as this is a massive benefit for the end user experience. It is mandatory for merchants using
+      ["Vipps Assisted Content Monitoring"](https://vippsas.github.io/vipps-developer-docs/docs/APIs/order-management-api/vipps-order-management-api#vipps-assisted-content-monitoring).
+
+
+- [ ] Correctly handle callbacks from Vipps, both for successful and unsuccessful payments.
```
Comment (rebekaburnett) [inline vipps-epayment-api-checklist.md:40]: Is this the same as "notification webhooks"?

File: api-guide/how-to-setup-notification-webhooks.md [hunk 2]
```diff
@@ -11,9 +11,6 @@ END_METADATA -->
 
 Notification webhooks can be setup in the Merchant Portal.
```
Comment (tht13) [inline api-guide/how-to-setup-notification-webhooks.md:12]: I think we should actually mark this file as draft and hidden, things here are changing slightly with how the webhooks API is coming along

File: vipps-epayment-api-checklist.md [hunk 3]
```diff
@@ -1,6 +1,113 @@
...
+      Vipps does not have capacity to manually do this for you.
+
+
+  - [ ] Callback [`POST:[callbackPrefix]/v2/payments/{orderId}`](https://vippsas.github.io/vipps-developer-docs/api/ecom#tag/Merchant-Endpoints/operation/transactionUpdateCallbackForRegularPaymentUsingPOST)
+- [ ] Make sure to log and handle all errors.
+       All integrations should to display errors in a way that the user can see and understand them.
+- [ ] Avoid Integration pitfalls
+  - [ ] The Merchant _must not_ rely on `fallback` or `callback` alone, and must poll Payments Details [`GET:/epayment/v1/{reference}`][get-payment-endpoint].
+        as documented (this is part of the first item in this checklist, but it's still a common error). For pure payment status polling, the ePayment API is recommended.
+  - [ ] The merchant must handle that the `fallback` URL is opened in the default browser on the phone,
```
Comment (tht13) [inline vipps-epayment-api-checklist.md:53]: this property is now called returnUrl

File: vipps-epayment-api-checklist.md (hunk 1 above)
Comment (rebekaburnett) [reply vipps-epayment-api-checklist.md:40]: ok!

File: vipps-epayment-api-checklist.md [hunk 4]
```diff
@@ -1,6 +1,113 @@
...
+   [Vipps på Nett](https://www.vipps.no/produkter-og-tjenester/bedrift/ta-betalt-paa-nett/ta-betalt-paa-nett/).
+2. Vipps completes customer control (KYC, PEP, AML, etc).
+3. The merchant receives an email from Vipps saying that they can log in with
+   BankID on
+   [portal.vipps.no][portal-url]
+   and retrieve API keys.
+4. The merchant completes all checklist items above.
+   Please double check to avoid mistakes.
+5. The merchant verifies the integration in the test environment by checking that
+   there are test IDs (`orderId`) in the
```
[the remark this answers is not in the document]
Comment (rebekaburnett) [reply vipps-epayment-api-checklist.md:78]: ok

**Review by tht13 — APPROVED** [review]

File: api-guide/how-to-setup-notification-webhooks.md (hunk 2 above)
Comment (rebekaburnett) [reply api-guide/how-to-setup-notification-webhooks.md:13]: done

**Pull request merged**